AudioKit/Common/Internals/AudioKit.swift

```diff
@@ -28,7 +28,31 @@ public typealias AKCallback = () -> Void
     // MARK: - Internal audio engine mechanics
 
     /// Reference to the AV Audio Engine
-    @objc open static var engine = AVAudioEngine()
+    @objc open static var engine = AVAudioEngine() {
```

Comment:
Note: didSet will only be called in the case of `engine = myEngine`, it won't be called on init.  I wasn't aware that the engine was settable, if it wasn't a breaking change I would prefer that it be a let constant.  Either that, or we manage all of the implications that are associated with dynamically changing the engine.

Reply:
(Sorry for the delay here.) I'm not sure about this one. I know an engine *can* be released but I'm not sure how or if AK would handle someone actually doing this. One reason to potentially release an engine is if it's `input` node was touched to do mic analysis (via a get or set), which would shut down the taptic engine on devices with them; the only way I've found so far to nil out the input node once touched is to annihilate the engine by nil-ing it out.